composer.json

```
@@ -9,7 +9,7 @@
         }
     ],
     "require": {
-        "sylius/shop-bundle": "^1.7"
+        "sylius/shop-bundle": "^1.9"
```

```suggestion
        "sylius/shop-bundle": "^2.1.1"
```
Like https://github.com/Sylius/Sylius/blob/cd258706066ba9c20027f4c1616f33233d342fb3/composer.json#L70

---

Well, the line you indicate @oallain is for `sylius/theme-bundle`, not for `shop-bundle` :sweat_smile:

---

Damnit, sorry you have right 🤦🏼 .
So it's good for me.